docs/content/guides/dagster-pipes.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Dagster Pipes | Dagster Docs
+description: "Dagster Pipes provides a protocol between the orchestration environment (Dagster) and external execution (ex: Databricks) and a toolkit for building implementations of that protocol."
+---
+
+# Dagster Pipes
+
+At a high level, Dagster Pipes allows you to send Dagster context data into an external process which is then sent back to Dagster.
+
+For example, let’s say you want to incorporate some business logic in a Dagster pipeline. However, it needs to deal with pre-existing data pipelines and Python environments, it’s written in a language other than Python, or you wish to complete separate the orchestration and business logic execution environment.,
+
+Using Dagster Pipes, you can write a few lines of code in your business logic in an external environment and send metadata and event logs back to Dagster.
+
+---
+
+## Benefits
+
+With Dagster Pipes, you can:
+
+- Incorporate existing code into Dagster without huge refactors
+- Onboard stakeholder teams onto Dagster incrementally
+- Run code in external environments (with minimal dependencies) and stream log and structured metadata back to Dagster
```

Comment:
Run code in external environments and easily pass parameters and context to that code, and then stream unstructured logs and structured metadata back to Dagster